mentorApp/ios/Runner/GoogleService-Info.plist

```diff
@@ -0,0 +1,36 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
[Looking at step 3.2 in the instructions to add firebase to a flutter app](https://firebase.google.com/docs/flutter/setup?platform=ios), it says to add this file in the `Runner/Runner` directory. Was it not working when you did that?

Reply:
Using Xcode to do it works for me.

Reply:
Wait, so did they have the `Runner/Runner` directory??

Reply:
YES

Reply:
Open the iOS part of the whole project in the Xcode.

Reply:
Oh, ok! Should we move this file to that directory instead? It says in the instructions to move it to `Runner/Runner` and we currently have it in `/Runner`.

Reply:
As discussed offline, when we open xCode, we see that this file is located in `Runner/Runner`. Good job Rex! 👍

Reply:
Thank you Helen!